.gitignore

```diff
@@ -12,12 +12,11 @@ target/
 # jetbrains ide files
 *.idea
 
+# tracing flamegraph for performance benchmarking
+*.svg
+tracing.folded
+
 # where account and keys are stored
 *.json
 # except for example json file used for testing
 !nomnomnom.testnet.json
-!37cb5b936de257d014e663663b186e58e4c3c4c9b4c7643e7b89821e789b0b4b.json
-!b278b7752b844a0cc88f8dbdb053b860d6283fb353e7a317b3b82efe01396c3e.json
-!df86e9701801e30cf0649b2ea43aeacef60e394e076709d4519880d5a9c7bc9e.json
-!e4a6dd159d45d4104679e709d29d49a533e5df9467b311c9dbea7fb8f0077335.json
-!d7adc371186bc2013ec5de36be27ed9a36022af1112f7c62530baf6042690f61.json
```

Comment:
Just double-checking: are these key files supposed to be included in the repo? They contain private (testnet) keys. It appears some of them may be duplicated in `account_keys/nomnomnom.testnet.json`.

Reply:
They are supposed to be included in the repo as they are all testnet keys. The purpose is to explicitly show the format necessary for others and to make it easy for running out of the box.

Reply:
They are now moved into the nomnomnom.testnet.json file as one of the refactors done in this release is moving away from multiple individual key files, which can be hard to manage for deployments, to a single list of keys in a single file.